include/NovelRT/Ink/Story.h

```diff
@@ -0,0 +1,42 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_INK_STORY_H
+#define NOVELRT_INK_STORY_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Ink {
+  class Story {
+  private:
+    struct Exports {
+      intptr_t(*CreateFromJsonString)(const char* jsonString);
+      void(*ChooseChoiceIndex)(intptr_t storyHandle, int choiceIdx);
+      const char*(*Continue)(intptr_t storyHandle);
+      const char*(*ContinueMaximally)(intptr_t storyHandle);
+      bool(*GetCanContinue)(intptr_t storyHandle);
+      void(*ResetState)(intptr_t storyHandle);
+    };
```

Comment:
Minor nit, same as previous: Could we not use std::add_pointer<T>::type here to make these fnptrs easier to read?